config/smartyfront.config.inc.php

```diff
@@ -79,7 +79,7 @@ function withWidget($params, callable $cb)
 function smartyWidget($params, &$smarty)
 {
     return withWidget($params, function ($widget, $params) {
-        return $widget->renderWidget(null, $params);
+        return Hook::coreRenderWidget($widget, null, $params);
     });
```

Comment:
Can you please describe your use case? Why you need that need that change precisely?
As the call to Hook is actually an alias to the previous version, I would rather keep the current version until we have a good reason to change it.

```php
public static function coreRenderWidget($module, $hook_name, $params)
{
    return $module->renderWidget($hook_name, $params);
}
```

Reply:
Regarding the diff, that was an alias.
Anyway, I validate this PR as it seems nothing can be overridden for a widget displayed from a template.